protos/spike/generated/spike/spike_grpc_pb.d.ts

```diff
@@ -0,0 +1,58 @@
+// package: spike
```

Comment:
changed that, the proto file doesn't compiled  on each run of the server because if the proto will update in the server, it will cause an unimplemented error.